inscriptions/ISic020480.xml

```diff
@@ -0,0 +1,224 @@
+        <!--
+  <surface type="front">
+    <graphic n="screen" url="{concat('ISic', $num, '_tiled.tif')}"
+      height="4912px" width="7360px">
+      <desc>Add Description Here</desc>
+    </graphic>
+    <graphic n="print" url="{concat('ISic', $num, '.jpg')}" height="3680px" width="5520px">
+      <desc>Add Description Here</desc>
+    </graphic>
+  </surface>
+
+    -->
+    </facsimile>
+    <text>
+        <body>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
+                <ab>
+                    <lb n="1"/><supplied reason="subaudible">λύχνος</supplied> κοινὸς το͂ <gap reason="lost" extent="unknown" unit="character"/>
+                </ab>
+            </div>
+            <div type="apparatus">
+                <listApp>
+                    <app>
+                        <note>Text after Arena (IGASM 2)</note>
+                    </app>
+                </listApp>
+            </div>
+            <div type="translation" xml:lang="en" resp="#VM">
+                <p>(I am the lamp) shared between ...</p>
+            </div>
+            <div type="translation" xml:lang="it" resp="#VM">
+                <p>(Sono la lucerna) condivisa tra ...</p>
+            </div>
+            <div type="commentary" xml:lang="en" resp="#VM">
+                <p>In this partly fragmentary inscription the sense of κοινά could be, as has been noted 
+                    (cf. Dubois IGDS no.144d; Arena IGASM no.33), that of 'offering', of '(oil lamp) shared with' 
+                    (cf. ISic020443) or it could correspond to the feminine Κοίνα of the uncommon anthroponym Κοινός 
+                    (cf. LGPN V3a-32169; V3b-12961; V4-30188-89-90-91-92).</p> <!-- would it be a good solution to mention the LGPN entries via the digital version ids? -->
+            </div>
+            <div type="bibliography">
+                <listBibl type="edition">
+                    <bibl type="corpus" n="IGASM2[2]">
+                        <author>Arena</author>
+                        <date>2002</date>
+                        <citedRange>33, pl. XIII.2</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/65P7AM6N"/>
+                    </bibl>
+                    <bibl type="corpus" n="IGDS1">
+                        <author>Dubois</author>
+                        <date>1989</date>
+                        <citedRange>144d</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/RWXNUFEX"/>
+                    </bibl>
+                    <bibl type="bullettin" n="SEG">
+                        <citedRange>29.961</citedRange>
+                        <ptr target="http://zotero.org/groups/382445/items/F2GG87EQ"/>
+                    </bibl>
+                </listBibl>
+                <listBibl type="discussion">
+                    <bibl type="lexicon" n="LGPN"> <!-- check: can we add 'lexicon' as a category? -->
```

Comment:
LGPN information will be captured differently, by onomastic tagging etc. on the names in the text, etc., not in the bibliography